pkg/render/manager.go

```diff
@@ -555,6 +579,46 @@ func (c *managerComponent) managerProxyContainer() corev1.Container {
 	}
 }
 
+// managerContainer returns the manager container.
+func (c *managerComponent) managerPacketCaptureContainer() corev1.Container {
+	var volumeMounts []corev1.VolumeMount
+	if c.managementCluster != nil {
+		volumeMounts = append(volumeMounts, corev1.VolumeMount{Name: ManagerInternalTLSSecretCertName, MountPath: "/manager-tls", ReadOnly: true})
+	}
+
+	env := []v1.EnvVar{
+		{Name: "PACKETCAPTURE_API_LOG_LEVEL", Value: "Info"},
+	}
+
+	if c.keyValidatorConfig != nil {
+		env = append(env, c.keyValidatorConfig.RequiredEnv("PACKETCAPTURE_API")...)
+		volumeMounts = append(volumeMounts, c.keyValidatorConfig.RequiredVolumeMounts()...)
+	}
+
+	return corev1.Container{
+		Name:            "tigera-packetcapture",
+		Image:           c.packetCaptureImage,
+		LivenessProbe:   c.managerPacketCaptureLivenessProbe(),
+		Resources:       c.resourceRequirements(),
+		SecurityContext: podsecuritycontext.NewBaseContext(),
+		Env:             env,
+		VolumeMounts:    volumeMounts,
+	}
+}
+
+func (c *managerComponent) resourceRequirements() v1.ResourceRequirements {
```

Comment:
Where did these resource requirement values come from? Are they for the entire manager or just the packetcapture component?

The function name implies the entire manager should receive these.

Reply:
They are being applied for a single container.

Reply:
we should probably rename the function to indicate that then. 

I'm also interested in how you came to these numbers.

Reply:
I simply added the values that were in the K8s example. I do not have a very scientific approach to this.

Reply:
Talked offline: Removing the resource requirements.